tests/query/test_query_ast.py

```diff
@@ -224,6 +225,7 @@ def test_alias_validation(
     events = get_dataset("events")
     query = parse_query(query_body, events)
     request = Request("", query, HTTPRequestSettings(), {}, "")
+    assert isinstance(request.query, LogicalQuery)
```

Comment:
This code does not depend on request anymore. Removing the request entirely is easier than adding the assertion.